test/sample-format-records.json

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
this is a bit anti-pattern in the unit tests, but I felt like creating dedicated jsons for unit test groups, rather than sql operations makes more sense, since less unit tests need to be updated in case something changes.